src/render_tree.rs

```diff
@@ -152,7 +172,7 @@ pub struct Node {
 impl Node {
     pub fn new() -> Self {
         Self {
-            node_type: NodeType::Root,
+            node_type: NodeType::Root(true),
```

Comment:
This is a dummy value that needs to be included otherwise it seems like it's ignored going from Rust -> C